plugin/plugin.go

```diff
@@ -197,6 +227,11 @@ func createDeployment(args Args, payload Payload, token string) error {
 		return err
 	}
 	defer res.Body.Close()
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Response from Jira:%d\n%s\n", res.StatusCode, string(body))
```

Comment:
I'd recommend using logrus here, so that we only print the body when debug logging is enabled.  You can also use structured logging for status code, etc.

Reply:
Instead of `io.ReadAll` we could do something like this:

```
switch args.Level {
case "debug", "trace":
  out, _ := httputil.DumpResponse(resp, true)
  logrus.WithField("status", resp.Status).WithField("endpoint", endpoint).Debug("request complete")
  println(string(out))
}
```

I would probably avoid placing the body in logrus because it will be escaped into a single line, and therefore unreadable. The reason I would use `httpdump` is because it enables us to read the response body as a stream in most cases, unless debug or trace logging is enabled, in which case it reads the full response into memory.